python-json-rbac/config.py

```
@@ -2,16 +2,107 @@
 Centralised configuration – reads from environment variables or sane defaults.
 """
 import os
-from typing import Optional
+import re
+import secrets
+import hashlib
+from typing import Optional, List
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)